rxandroidble/src/main/java/com/polidea/rxandroidble2/internal/serialization/FIFORunnableEntry.java

```diff
@@ -35,44 +35,46 @@ public int compareTo(@NonNull FIFORunnableEntry other) {
     }
 
     public void run(QueueSemaphore semaphore, Scheduler subscribeScheduler) {
+
+        if (operationResultObserver.isDisposed()) {
+            RxBleLog.d("Operation was about to be run but the observer was already disposed: " + operation);
```

Comment:
How about?

The operation was about to be run but the observer had been already disposed